### PR TITLE
Add implementation template images

### PR DESCRIPTION
**Pull request by ValeriiMalkov**
Add implementation template images
on the HMI for next RPC:

- SetGlobalProperties
- AddCommand
- Show
- ShowConstantTBT
- SendLocation
- UpdateTurnList
- Alert
- ScrollableMessage
- AlertManeuver
- CreateInteractionChoiceSet

### CODE REVIEW

File: app/controller/sdl/Abstract/Controller.js [hunk 1]
```diff
@@ -511,7 +516,12 @@ SDL.SDLController = Em.Object.extend(
       switch (element.groupName) {
         case 'AlertPopUp':
         {
+          if(this.isWarning!='WARNINGS'){
```
Comment (BSolonenko) [inline app/controller/sdl/Abstract/Controller.js]: @ValeriiMalkov  the constant value must be on the left.

File: app/model/sdl/MediaModel.js [hunk 2]
```diff
@@ -363,6 +385,7 @@ SDL.SDLMediaModel = SDL.ABSAppModel.extend({
         this.appInfo.set('customPresets.' + i, params.customPresets[i]);
       }
     }
+
```
Comment (BSolonenko) [inline app/model/sdl/MediaModel.js]: @ValeriiMalkov please delete this diff.

File: app/view/media/sdl/controllsView.js [hunk 3]
```diff
@@ -39,6 +39,7 @@ SDL.SDLMediaControlls = Em.ContainerView.create(
       'Controls',
       'tuneButtons'
     ],
+
```
Comment (BSolonenko) [inline app/view/media/sdl/controllsView.js]: @ValeriiMalkov please delete this diff.

File: app/view/media/sdl/controllsView.js [hunk 4]
```diff
@@ -48,6 +49,7 @@ SDL.SDLMediaControlls = Em.ContainerView.create(
         elementId: 'app_view_info',
         template: Em.Handlebars
           .compile(
+
```
Comment (BSolonenko) [inline app/view/media/sdl/controllsView.js]: @ValeriiMalkov please delete this diff.

File: app/view/sdl/AlertPopUp.js [hunk 5]
```diff
@@ -122,7 +122,11 @@ SDL.AlertPopUp = Em.ContainerView.create(
         SDL.SDLController.alertResponse(
           SDL.SDLModel.data.resultCode['ABORTED'], this.alertRequestId
         );
-      } else {
+      }else if(reason=='WARNINGS'){
```
Comment (BSolonenko) [inline app/view/sdl/AlertPopUp.js]: @ValeriiMalkov  the constant value must be on the left.

File: app/view/home/controlButtons.js [hunk 6]
```diff
@@ -53,8 +53,74 @@ SDL.ControlButtons = Em.ContainerView.create({
...
+      SDL.InfoView.leftMenu.items.appsButton.setMode(this.selection);
+      SDL.InfoView.leftMenu.items.calendarButton.setMode(this.selection);
+      SDL.InfoView.leftMenu.items.goToCD.setMode(this.selection);
+      SDL.InfoView.leftMenu.items.travelLinkButton.setMode(this.selection);
+      SDL.InfoView.leftMenu.items.sdlButton.setMode(this.selection);
+      SDL.TurnByTurnView.nextTurnIconImage.setMode(this.selection);
+      SDL.TurnByTurnView.turnIconImage.setMode(this.selection);
+      SDL.InteractionChoicesView.set('imageMode',this.selection);
+      SDL.InteractionChoicesView.updateIcons();
+      if(SDL.SDLController.model!=null){
```
Comment (BSolonenko) [inline app/view/home/controlButtons.js]: @ValeriiMalkov  the constant value must be on the left.

File: app/view/sdl/shared/turnByTurnView.js [hunk 7]
```diff
@@ -170,36 +185,38 @@ SDL.TurnByTurnView = SDL.SDLAbstractView.create(
...
+          'this.parentView.isTemplateNextTurnIcon:is-template'
         ],
-        style: function() {
-          if (this._parentView.nextTurnIcon) {
-            return 'background-image: URL(' + this._parentView.nextTurnIcon +
-              ');';
+        templateName: 'rightTextOverLay',
+        iconBinding: 'getIcon',
+        getIcon: function() {
+          if (this._parentView != null && this._parentView.nextTurnIcon) {
```
Comment (BSolonenko) [inline app/view/sdl/shared/turnByTurnView.js]: @ValeriiMalkov  the constant value must be on the left.

File: css/general.css [hunk 8]
```diff
@@ -826,10 +868,12 @@ margin-top: 90px;
...
     float: left;
     width: 50px;
-    position: relative;
-
+    position: absolute;
+    left:inherit;
 }
 
+
+
```
Comment (BSolonenko) [inline css/general.css]: @ValeriiMalkov please delete this diff.

File: ffw/UIRPC.js [hunk 9]
```diff
@@ -67,22 +67,22 @@ FFW.UI = FFW.RPCObserver.create(
     /**
      * connect to RPC bus
      */
-    connect: function() {
+    connect: function () {
```
Comment (BSolonenko) [inline ffw/UIRPC.js]: @ValeriiMalkov please delete this diff.

File: app/controller/SettingsController.js [hunk 10]
```diff
@@ -341,21 +341,12 @@ SDL.SettingsController = Em.Object.create(
...
-              })
-            } else {
-              FFW.BasicCommunication.OnSystemRequest(
-                'PROPRIETARY',
-                SDL.SettingsController.policyUpdateFile,
-                SDL.SDLModel.data.policyURLs[0].url,
-                SDL.SDLModel.data.policyURLs[0].appID
-              );
-            }
+            FFW.BasicCommunication.OnSystemRequest(
```
Comment (yang1070) [inline app/controller/SettingsController.js]: not related to template images

File: app/controller/sdl/Abstract/Controller.js [hunk 11]
```diff
@@ -50,6 +50,7 @@ SDL.SDLController = Em.Object.extend(
      * Active application model binding type {SDLAppModel}
      */
     model: null,
+    isWarning:false,
```
Comment (yang1070) [inline app/controller/sdl/Abstract/Controller.js]: is the warning stuff relate to template images?

File: ffw/UIRPC.js [hunk 12]
```diff
@@ -136,228 +136,812 @@ FFW.UI = FFW.RPCObserver.create(
...
-            // SDL.SDLModel.data.resultCode["WARNINGS"]; } else { If no
-            // available data sent error response and stop process current
-            // request this.sendError(this.errorResponsePull[request.id].code,
-            // request.id, request.method, "Unsupported " +
-            // this.errorResponsePull[request.id].type + " type. Request was
-            // not processed."); this.errorResponsePull[request.id] = null;
-            // return; } }
-            if (SDL.SDLModel.onUIAlert(request.params, request.id)) {
-              SDL.SDLController.onSystemContextChange(request.params.appID);
+              // Verify if there is an unsupported data in request
```
Comment (yang1070) [inline ffw/UIRPC.js]: why keep these commented out code?

File: app/controlls/Button.js [hunk 13]
```diff
@@ -165,6 +180,11 @@ SDL.Button = Em.View.extend(Ember.TargetActionSupport,
         '<img {{bindAttr class="view.icon:ico"}} {{bindAttr src="view.icon"}} />' +
         '<span class="right_text">{{view.text}}</span>'
       ),
+      rightTextOverLay: Em.Handlebars.compile(
```
Comment (yang1070) [inline app/controlls/Button.js]: does softbutton in alert support showing image?

**Pull request merged**